### DISCUSSION TITLE
whiteboard customization

### DISCUSSION THREAD

**Discussion by ddd793** [mynaparrot/plugNmeet-server#554] [General]
I am using the plugNmeet in a virtual machine running Docker containers (redis, mariadb, livekit, livekit-ingress, plugnmeet-api) and client.zip from release page.

I need to add a new button with a new function to the whiteboard on the server. I have downloaded the client code from the plugnmeet-client repository to my PC (git clone, npm install, npm start) and created two files: test.tsx and test.css, in the client/src/components/test folder. I have also modified the index.tsx file in the client/src/components/whiteboard folder.
If I open the login page at http://localhost:3000, I can see my new button an It is works.

How can I transfer the changes from my local machine to the server so that the button works? 
The file client.zip contains only the assets folder, and there are no other folders like components.

**Comment by jibon57** [comment]
You'll need to run `npm run build`. That will build everything inside `dist` directory. Later you can transfer files of that directory to your server.

**Reply by jibon57** [reply]
https://github.com/mynaparrot/plugNmeet-client/blob/f5c3ca3af8c72ab64b172d40b7e5fc5139066a64/.github/workflows/release-client.yml#L15

**Reply by ddd793** [reply]
Thank you very much, it works.
